app/store/reducers/healthcareAuthoritiesReducer.ts

```diff
@@ -42,7 +45,20 @@ const healthcareAuthoritiesReducer = createReducer(initialState, (builder) =>
     .addCase(getHealthcareAuthorities_failure, (state, { meta: { error } }) => {
       state.request.status = ApiStatus.FAILURE;
       state.request.errorMessage = error.message;
-    }),
+    })
+    .addCase(
+      toggleSelectedHealthcareAuthorityAction,
+      (state, { payload: { authority, overrideValue } }) => {
```

Comment:
is this something that needs to be in the global state? could it stay local to the component?

Reply:
yeah, this will need to be consumed by the background running task to determine intersections